tox.ini

```diff
@@ -2,7 +2,7 @@
 envlist = py35-django22,py38-django{22,30}
 
 [doc8]
-max-line-length = 120
+ignore = D001
```

Comment:
Please add the comment you see here: https://github.com/edx/code-annotations/blob/master/tox.ini#L4-L6